### PR TITLE
WF-577: Demonstrate our snapshot conversion workflow

### PR DESCRIPTION
**Pull request by JonasImprobable**
Created a `convert_snapshot.sh` script for showing how to use the snapshot converter.
Updated our documentation.

### CODE REVIEW

File: SpatialOS/scripts/convert_snapshot.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# This script converts a binary snapshot file to its text format representation
+# Usage: convert_snapshot.sh <binary snapshot file name, optional, default value: "default.snapshot"> <text snapshot file name, optional, default value: "default.txt">
```
Comment (frozenL) [inline SpatialOS/scripts/convert_snapshot.sh]: The `optional`s look confusing to me - if I understand the script correctly, you can't pass `text snapshot file name` without passing `binary snapshot file name`, so in a way, it's not optional? 🤔

Comment (JonasImprobable) [reply SpatialOS/scripts/convert_snapshot.sh]: Discussed offline.

File: SpatialOS/snapshots/default.txt [hunk 2]
```diff
@@ -0,0 +1,106 @@
+entity {
```
Comment (frozenL) [inline SpatialOS/snapshots/default.txt]: considering that this file is generated, is it necessary to include it?

Comment (JonasImprobable) [reply SpatialOS/snapshots/default.txt]: I think it's fine either way. I found it a neat idea to include this file here for "demonstration purposes". Happy to remove it if people feel strongly about it.

Comment (frozenL) [reply SpatialOS/snapshots/default.txt]: I don't feel strongly either way. If we decide to keep it as an example, how about documenting this example file in `docs/migration-guide/faq.md`?

Comment (JonasImprobable) [reply SpatialOS/snapshots/default.txt]: Good idea. Done.

**Pull request merged**